locallib.php

```diff
@@ -300,7 +300,8 @@ function theme_boost_union_get_staticpage_link($page) {
  */
 function theme_boost_union_get_staticpage_pagetitle($page) {
     // Get the configured page title.
-    $pagetitleconfig = get_config('theme_boost_union', $page.'pagetitle');
+    $pagetitleconfig = format_string(get_config('theme_boost_union', $page.'pagetitle'), false,
+    ['context' => \context_system::instance()]);
```

Comment:
It took me some seconds to understand that using the system context is correct here - we do not want to break the multilanguage links to the static pages in the footer of a course page just because a teacher disabled the multilang filter in his course.

Maybe you want to add this a comment to the code?

Reply:
Pardon me, @abias. I hat the CHANGES.md ready but forget to add it to the commit. And I do always interpret striplinks the wrong way around.

Reply:
The comment-to-code ratio is quite high in our plugins and comments do not cost anything. But well, it's your patch. Let keep it as it is.